app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/PaginationUtil.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tariffclassificationfrontend.views.partials
+
+import scala.collection.immutable.Seq
+
+object PaginationUtil {
```

Comment:
![](https://media.giphy.com/media/aZ3LDBs1ExsE8/giphy.gif)

Reply:
It's to generate the `Previous 1 2 3 4 5 6 Next`

Reply:
I tried various strategies, ended up going with this... :(